app/views/charges/new.html.erb

```diff
@@ -1,4 +1,15 @@
-<%= form_tag charges_path do %>
-  <h4>Click the button!</h4>
-  <script class='stripe-button' src="https://checkout.stripe.com/checkout.js" data-key="<%= @stripe_btn_data[:key] %>" data-amount=<%= @stripe_btn_data[:amount] %> data-description="<%= @stripe_btn_data[:description] %>" ></script>
-<% end %>
+<div class="container">
+  <% if current_user.premium? %>
+    <%= button_to 'Downgrade Membership', downgrade_path, class: 'btn' %>
```

Comment:
This looks right except you need to figure out the correct path method.  It might be downgrade_path, or it might be something else (see my comment below).